src/engine/controls/cuecontrol.h

```diff
@@ -346,6 +351,8 @@ class CueControl : public EngineControl {
     ControlObject* m_pHotcueFocusColorNext;
     ControlObject* m_pHotcueFocusColorPrev;
 
+    ControlProxy* m_pPassthrough;
```

Comment:
Unused?

Reply:
yes, because creating and connecting it obviously got lost when splitting cue control changes into separate commits.
so: No, I'll --fixup